src/sinks/amqp/sink.rs

```diff
@@ -43,6 +43,7 @@ impl AmqpSink {
             .await
             .map_err(|e| BuildError::AmqpCreateFailed { source: e })?;
 
+        // Enable confirmations on the channel.
```

Comment:
This only needs to be called once: https://www.rabbitmq.com/confirms.html#publisher-confirms

Reply:
good find!